src/Core/composer.json

```diff
@@ -32,12 +32,11 @@
         "ext-json": "*",
         "ext-openssl": "*",
         "acmephp/ssl": "^2.0",
-        "guzzlehttp/guzzle": "^6.0|^7.0",
-        "guzzlehttp/psr7": "^1.7|^2.1",
+        "guzzlehttp/guzzle": "^7.2",
+        "guzzlehttp/psr7": "^2.4.5",
         "lcobucci/jwt": "^5.3",
-        "psr/http-client": "^1.0",
```

Comment:
Did you intentionally remove `psr/http-client`?

Reply:
yes, it's not used at all